pkg/util/nativeutils/eventbus/listenermap.go

```diff
@@ -36,8 +36,10 @@ func (h *listenerMap) Store(key topics.Topic, value Listener) uint32 {
 func (h *listenerMap) Load(key topics.Topic) []idListener {
 	h.lock.RLock()
 	listeners := h.listeners[key]
+	dup := make([]idListener, len(listeners))
+	copy(dup, listeners)
 	h.lock.RUnlock()
```

Comment:
could we add the unlock with defer on the begin of the func ? just for consistency.

Reply:
No, defer is not needed here.